two_body_scf_guess_III.py

```diff
@@ -0,0 +1,146 @@
+from wrap_pyscf import *
+import numpy as np
+from itertools import combinations
+
+def make_monomer_dens(monomers, bs):
+    monomer_dens = []
+    for j in monomers:
+        k = density(j, bs)
+        monomer_dens.append(k)
+    return monomer_dens
+
+def make_offset(a):
+    lst = []
+    lst.append(a-1)
+    for i, num in enumerate(lst):
+        if num > 1:
+            k = lst[-1]-1
+            lst.append(k)
+    lst1 = lst
+    count = 0
+    lst2 = []
+    for item in lst1:
+        count = count + item
+        lst2.append(count)
+    offset = lst2
+    return offset
+
+def dimer_lst(monomers, a, unit_guess):
```

Comment:
`unit_guess = 2` tells `itertools.combinations` to make dimers from monomers. If you want to call it something else I'm on board.

Reply:
Oh and both tests were passed.